integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/Hooks.java

```diff
@@ -39,7 +44,23 @@ public Hooks(TestContext context) {
         this.context = context;
     }
 
-    @BeforeAll
+    @BeforeAll(order = 0)
+    public static void initializeLocalTestConfig() {
+        final String filename = "localconfig.properties";
+        File file = new File(filename);
+        if (file.exists()) {
+            try {
+                InputStream inputStream = new FileInputStream(file);
+                Properties prop = System.getProperties();
+                prop.load(inputStream);
```

Comment:
I'm not sure whether it is good to prioritize localconfig.properties over the system properties. I would suggest to do it the opposite way. To prioritize system properties over localconfig.properties. It will allow us to rewrite some parameters from localconfig.properties on command line etc.

Reply:
I think going forward there may be possibilities to introduce System properties for different tests..So its good to keep localconfig.properties file to define the parameters..
@sutaakar wdyt?

Reply:
In the future when we start leveraging RHOS implementation for remote systems then we should switch to their approach of defining properties using properties file.
So IMHO we don't need to think up some complex implementations and go with what we have here, as it will be replaced/ported once we migrate.